lib/vm.js

```diff
@@ -54,4 +54,11 @@ exports.runInThisContext = function(code, options) {
   return script.runInThisContext(options);
 };
 
+exports.runInGlobalContext = function(code, options) {
+  var script = new Script(require('module').wrap(code), options);
+  return script.runInThisContext(options)(exports, require, module,
+                                                            __filename,
+                                                            __dirname);
```

Comment:
Renaming would be fine for me. I am just looking for ways to have similar `require` behaviour.